services/gundeck/src/Gundeck/Push/Data.hs

```diff
@@ -49,10 +49,10 @@ insert u t a p e o c = retry x5 $ write q (params LocalQuorum (u, t, a, p, e, o,
     q = "insert into user_push (usr, transport, app, ptoken, arn, connection, client) values (?, ?, ?, ?, ?, ?, ?)"
 
 updateArn :: MonadClient m => UserId -> Transport -> AppName -> Token -> EndpointArn -> m ()
-updateArn uid transport app token arn = retry x5 $ write q (params LocalQuorum (arn, uid, transport, app, token))
+updateArn uid transport app token arn = retry x5 . void $ trans q (params LocalQuorum (arn, uid, transport, app, token))
```

Comment:
I guess ARN without connectionId and clientId is not valid?

Reply:
i would think so, yes.